django/db/migrations/state.py

```diff
@@ -231,11 +231,11 @@ class AppConfigStub(AppConfig):
     path = ''
 
     def __init__(self, label):
-        self.label = label
         # App-label and app-name are not the same thing, so technically passing
         # in the label here is wrong. In practice, migrations don't care about
         # the app name, but we need something unique, and the label works fine.
-        super().__init__(label, None)
+        self.label = label
+        self.name = label
```

Comment:
I wonder does it still make sense to subclass `AppConfig`, we only want to have a dict of models for each app. Subclassing `AppConfig` looks unnecessary :thinking: @charettes Can I ask for your opinion?

Reply:
I agree if it doesn't cause to much pain in refactoring. It looks like `StateApps(Apps)` might be relying on other properties than `.label` and `.name`

Reply:
Yeah, I've checked and it's probably not worth to change this, thanks.